### ISSUE TITLE
pytorch conda channel is missing torchvision `0.2.2`

### ISSUE THREAD

**Issue by pmeier** [pytorch/vision#7456]
After creating and discussing a question about this on [https://github.com/conda-forge/torchvision-feedstock](https://github.com/conda-forge/torchvision-feedstock), I have verified on an earlier image that torchvision 0.2.2 was actually installed _from the pytorch channel_, and not the conda-forge channel.  As a result, I've returned to this issue to follow up on the problem.  The pytorch channel is also missing torchvision `0.2.2`:

```
$ conda search -c pytorch torchvision | grep '0\.2'
torchvision                    0.2.0          py27_0  pkgs/main           
torchvision                    0.2.0          py35_0  pkgs/main           
torchvision                    0.2.0          py36_0  pkgs/main           
torchvision                    0.2.1          py27_0  pkgs/main           
torchvision                    0.2.1          py35_0  pkgs/main           
torchvision                    0.2.1          py36_0  pkgs/main           
torchvision                    0.2.1          py37_0  pkgs/main 
```

_Originally posted by @skroon in https://github.com/pytorch/vision/issues/4665#issuecomment-1482448245_

**Comment by taceysa** [comment]
Hello, I was wondering if there were any updates on this? I am trying to install a conda environment (https://github.com/MatthewMasters/EDM-Dock/) which includes torchvision==0.2.2=py_3 in the packages, but it cannot find the package.

Thanks!

**Comment by malfet** [comment]
Some older packages were deleted/archived to make space for a new release, but now that storage issues has been resolved, I guess we can bring them back now. 
cc: @seemethere , @atalman

**Comment by atalman** [comment]
I fully agree about pytorch-archive channel we should probably invest in migrating our complete backups in this channel. As a cheap option can we just expose the s3 download link to these backed up packages ? If people want to access these packages, they can probably just download it and install from local file ?? 
cc: @malfet , @seemethere

**Comment by pmeier** [comment]
> Some older packages were deleted/archived to make space for a new release

This seems fine to me, but I find it quite weird that we dropped `0.2.2`, while `0.2.0` and `0.2.1` are still available.

**Comment by atalman** [comment]
I don't see them available on official channel:
<img width="924" alt="Screenshot 2023-04-03 at 4 55 15 PM" src="https://user-images.githubusercontent.com/7563158/229625825-d03cd3a3-eefd-4fa2-bd6d-0a0e1ebf64a5.png">

**Comment by pmeier** [comment]
See my top comment for a search from CLI:

```
$ conda search -c pytorch torchvision | grep '0\.2'
torchvision                    0.2.0          py27_0  pkgs/main           
torchvision                    0.2.0          py35_0  pkgs/main           
torchvision                    0.2.0          py36_0  pkgs/main           
torchvision                    0.2.1          py27_0  pkgs/main           
torchvision                    0.2.1          py35_0  pkgs/main           
torchvision                    0.2.1          py36_0  pkgs/main           
torchvision                    0.2.1          py37_0  pkgs/main
$ conda install --dry-run -c defaults -c pytorch python torchvision=0.2
[...]
```

**Comment by taceysa** [comment]
> Hello, I was wondering if there were any updates on this? Thanks!

Any updates on getting v0.2.2 back to being available?

**Comment by malfet** [comment]
@taceysa just curious, why do you need v0.2.2? It's bad to have v0.2.2, as it's a accidentally acts as placeholder (as a noarch package) for any incorrect installation command.

**Comment by taceysa** [comment]
I am trying to install an open-source software called EDM-Dock for one of our scientists at AbbVie. https://github.com/MatthewMasters/EDM-Dock

The install requires creating a conda environment using their environment.yaml file. The channels this yaml file calls are:
channels:
  - pyg
  - iopath
  - anaconda
  - conda-canary
  - omnia
  - rmg
  - conda-forge
  - rdkit
  - pytorch
  - defaults

And one of the dependencies is:

  - torchvision=0.2.2=py_3

Is there another version of torchvision that could be used seamlessly in place with v0.2.2? Or is there another way we can access this version?
